accname/name/comp_host_language_label.tentative.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Name Comp: Host Language Label</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    <!-- Ambiguity coming from Host Language Label in accname specs (https://www.w3.org/TR/accname-1.2/#comp_host_language_label) - PR that tries to solve it https://github.com/w3c/aria/pull/2405 -->
+    <h2>HTML input with label with role none</h2>
+    <h3>HTML input label/for</h3>
+    <label role="none" for="lrp">input label</label>
+    <input id="lrp" type="text" data-expectedlabel="input label" data-testname="html: label[for][role=none]" class="ex-label">
+    <h3>HTML label encapsulation</h3>
+    <label role="none">
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: label[role=none] encapsulation" class="ex-label">
+    </label>
+
+    <!-- This test case is specifically designed to check the accessible name, but a separate test case has been created in wai-aria/role/role_none_conflict_resolution.tentative.html to check the role -->
+    <h2>HTML input with role="none" and label associated (presentational roles conflict resolution)</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irn">input label</label>
+    <input id="irn" type="text" data-expectedlabel="input label" data-testname="html: label[for]input[role=none]" class="ex-label" role="none">
```

Comment:
Is the goal here that the `none` role is invalidated by the input's focusability? If so, say so in the comment or test name. (FYI, test name is not a valid CSS selector. IIRC, you'd need a `+` before `input` to match the adjacent sibling selector.) 

Perhaps `data-testname="html:label[for]+input[role=none] (role invalidated due to focusability)"`

Otherwise, if `none` is the expected role, I don't think this is a valid test, because there is no mandate in WebDriver to have a backing object on an ignored element or to return a label from the non-existent object; not even an empty string. If label from a 'none' role is the intention, I would remove this test and possible refile as a new "blocked by testability" issue.

Reply:
Thanks @cookiecrook!

I'm updating the data-testname with your suggestion since this scenario triggers the presentational roles conflict resolution, and the explicit role set is expected to be ignored due to focusability. Worth mentioning that, in this specific test case, we don't care about the role as it's meant to test only the accessible name. I'm currently trying to address the role topic in the other file: [wai-aria/role/role_none_conflict_resolution.tentative.html](https://github.com/web-platform-tests/wpt/pull/50002/files#diff-701a92ad8707877b8e9fabb34beaa6e85daa5c714bfb58587b40c0c6dabfe0c4).

Reply:
(of course I understand the impact of the role, since the acc name on generic is irrelevant)

Reply:
I've updated all data-testname values with valid CSS selectors for consistency and added explanations for why the label should be computed this way.

Reply:
Following up on the second point in the next review here: https://github.com/web-platform-tests/wpt/pull/50002#pullrequestreview-2587331996